setup.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: page
+title: Set up GUAC with Docker Compose
+permalink: /setup/
+nav_order: 2
+---
+
+
+# Set up GUAC with Docker Compose
+
+{: .note }
+If you’d prefer, you can set up [GUAC with Kubernetes](https://github.com/kusaridev/helm-charts/tree/main/charts/guac) instead.
```

Comment:
Just curious, what's the reasoning for having the Docker Compose setup option in the side bar, but not having the Kubernetes setup there? Why does Kubernetes link out to another repo?